src/Hl7.Fhir.Serialization.Shared.Tests/ParseDemoPatientLineInfoAnnotationPoco.cs

```diff
@@ -0,0 +1,87 @@
+﻿using FluentAssertions;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Utility;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections;
+using System.IO;
+
+namespace Hl7.Fhir.Serialization.Tests
+{
+    [TestClass]
+    public class ParseDemoPatientLineInfoAnnotationPoco
+    {
+        public T getXmlPocoAnnotated<T>(string xml) where T : Base
+        {
+            try
+            {
+                return new FhirXmlDeserializer(new DeserializerSettings(){ AnnotateLineInfo = true }).Deserialize<T>(xml);
```

Comment:
If you do  

`new DeserializerSettings() { AnnotateLineInfo = true }.UsingMode(Ostrich).Deserialize();`

Then you don't need the try/catch, since it will never throw and always return the partial result.